concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+Bowern-2021-203c-83	83	fish poison 	641	subsistence tool	Culture Vocabulary	348	FISH POISON
+Bowern-2021-203c-84	84	fishing line 	642	subsistence tool	Culture Vocabulary	19	FISHING LINE
+Bowern-2021-203c-85	85	fishtrap 	726	subsistence tool	Culture Vocabulary	389	FISH TRAP
+Bowern-2021-203c-86	86	fishtrap (permanent) 	782	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-87	87	fishtrap (woven) 	783	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-88	88	flat bread, cassava bread 	727	food	Culture Vocabulary		
+Bowern-2021-203c-89	89	flour (bush) 	785	food	Culture Vocabulary		
+Bowern-2021-203c-90	90	flour (European) 	814	food	Culture Vocabulary		
+Bowern-2021-203c-91	91	flour/meal from manioc 	728	food	Culture Vocabulary		
+Bowern-2021-203c-92	92	flute 	646	culture-mythology	Culture Vocabulary	777	FLUTE
+Bowern-2021-203c-93	93	gambling sticks 	647	other	Culture Vocabulary		
+Bowern-2021-203c-94	94	game animal 	648	food	Culture Vocabulary		
+Bowern-2021-203c-95	95	genipap 	167	flora-fauna	Culture Vocabulary		
+Bowern-2021-203c-96	96	genipap 	730	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-97	97	ghost (of dead person)/evil spirit 	634	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-98	98	glue 	650	subsistence tool	Culture Vocabulary	1069	GLUE
+Bowern-2021-203c-99	99	gourd (dipper/bowl) 	731	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-100	100	grater 	732	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-101	101	grave 	651	culture-mythology	Culture Vocabulary	1773	GRAVE
+Bowern-2021-203c-102	102	grid of sticks for smoking meat or placing objects (shelf) 	733	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-103	103	griddle for cooking flatbread 	734	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-104	104	grog 	786	food	Culture Vocabulary		
+Bowern-2021-203c-105	105	gun 	703	acculturation	Culture Vocabulary	1566	GUN
+Bowern-2021-203c-106	106	hair belt 	787	dress	Culture Vocabulary		
+Bowern-2021-203c-107	107	hammock 	736	culture-material	Culture Vocabulary	2019	HAMMOCK
+Bowern-2021-203c-108	108	hat 	704	dress	Culture Vocabulary	2130	HEADGEAR
+Bowern-2021-203c-109	109	headband 	788	dress	Culture Vocabulary	330	HEADBAND
+Bowern-2021-203c-110	110	healer 	789	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-111	111	hollow log/trough for beer-making 	738	narcotics	Culture Vocabulary		
+Bowern-2021-203c-112	112	honey 	655	food	Culture Vocabulary	942	HONEY
+Bowern-2021-203c-113	113	house 	656	other	Culture Vocabulary	1252	HOUSE
+Bowern-2021-203c-114	114	house (European) 	790	other	Culture Vocabulary		
+Bowern-2021-203c-115	115	hunt 	657	food	Culture Vocabulary	1435	HUNT
+Bowern-2021-203c-116	116	initiate boy 	658	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-117	117	initiate girl 	659	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-118	118	initiation (circumcision) 	791	culture-mythology	Culture Vocabulary		
```

Comment:
CIRCUMCISION ?

Reply:
Could be, but it seemed a bit too narrow to me, considering that circumcision is only added in parentheses.